ui/src/app/workflows/components/workflow-drawer/workflow-drawer.tsx

```diff
@@ -0,0 +1,75 @@
+                    <div className='workflow-drawer__labels--list'>
+                        <WorkflowLabels
+                            workflow={wf}
+                            onChange={key => {
+                                this.props.onChange(key);
+                            }}
+                        />
+                    </div>
+                </div>
+                {!wf.status.message ? null : (
+                    <div className='workflow-drawer__section workflow-drawer__message'>
+                        <div className='workflow-drawer__title workflow-drawer__message--label'>MESSAGE</div>
+                        <div className='workflow-drawer__message--content'>{wf.status.message}</div>
+                    </div>
+                )}
+                {!wf.status.resourcesDuration ? null : (
+                    <div className='workflow-drawer__section'>
+                        <div className='workflow-drawer__resourcesDuration'>
+                            <div className='workflow-drawer__title'>
+                                RESOURCES DURATION&nbsp;
+                                <a href='https://github.com/argoproj/argo/blob/master/docs/resource-duration.md' target='_blank'>
+                                    <i className='fas fa-info-circle' />
+                                </a>
+                            </div>
+                            <div className='workflow-drawer__resourcesDuration--container'>
+                                <div>
+                                    <span className='workflow-drawer__resourcesDuration--value'>{wf.status.resourcesDuration.cpu} sec</span>
+                                    <span className='workflow-drawer__resourcesDuration--label'>(*1 CPU)</span>
+                                </div>
+                                <div>
+                                    <span className='workflow-drawer__resourcesDuration--value'>{wf.status.resourcesDuration.memory} sec</span>
+                                    <span className='workflow-drawer__resourcesDuration--label'>(*100Mi Memory)</span>
+                                </div>
+                            </div>
+                        </div>
+                    </div>
+                )}
+                {!wf.status.conditions ? null : (
+                    <div className='workflow-drawer__section'>
+                        <div className='workflow-drawer__title'>CONDITIONS</div>
+                        <div className='workflow-drawer__conditions'>
+                            {wf.status.conditions.map(condition => {
+                                return (
+                                    <div className='tag' key={`${wf.metadata.namespace}-${wf.metadata.name}-${condition.type}-${condition.status}`}>
+                                        <div className='key'>{condition.type}</div>
+                                        <div className='value'>{condition.status}</div>
```

Comment:
With `Conditions`, the `Message` field is often more important than the status field. In fact, we only show the status field at all only if the message field is empty. Furthermore, these message fields can be quite long... take a look at this example 
![image](https://user-images.githubusercontent.com/614838/83785556-53149200-a646-11ea-9fe9-d3fb6882bb54.png)

So I'm not sure the `tag` class that we use to display "labels" would work here. What do you think?

Reply:
Ah I see. With context, I agree - when I first saw that example, I thought that the SpecWarning portion was not itself a condition, but a constant warning displayed for every workflow.